src/lex.js

```diff
@@ -61,6 +59,36 @@ function asyncTrigger() {
   };
 }
 
+/**
+ * Implementation of ES6 `String.fromCodePoint` for legacy environments
+ */
+var fromCodePoint = String.fromCodePoint || function(codepoint) {
+  if (codepoint <= 0xffff) { // BMP
+    return String.fromCharCode(codepoint);
+  } else { // astral
+    var cp = codepoint - 0x10000;
+    var highSurrogate = (cp >> 10) + 0xd800;
+    var lowSurrogate = (cp % 1024) + 0xdc00;
+    return String.fromCharCode(highSurrogate, lowSurrogate);
+  }
+};
+
+/**
+ * Implementation of ES6 `String#codePointAt(0)` for legacy environments
+ */
+function codePoint(char) {
```

Comment:
This is a bit inconsistent. Why not define this function in the same way as `fromCodePoint` and avoid the Istanbul-specific workaround?